mission/edit_me/briefing/employer.html

```diff
@@ -0,0 +1 @@
+INSERT NAME OF EMPLOYER HERE
```

Comment:
This might also need colored headers or a title (aside from the built-in one) if we want it to follow the design system in SITUATION. Have to come up with a way to slap one on here.

Reply:
It should just be a single word / line though, the title is the header, unless we want to remove employer and put it under another tab

Reply:
I get it, however, it is largely inconsistent in design when you go from SITUATION to other entries. I'd add it into another tab then.